medcat-service/medcat_service/types.py

```diff
@@ -1,9 +1,33 @@
-from typing import Literal, TypedDict
+from datetime import datetime
+from typing import List, Literal, Optional, Union
 
+from pydantic import BaseModel
 
-class HealthCheckResponse(TypedDict):
+BaseModel.model_config['protected_namespaces'] = ()
```

Comment:
The way it's done here is a global change. It would affect every other `pydantic` model that inherits from `BaseModel` (and doesn't explicitly set protected namespaces), including ones in dependencies.
I don't think that's what we want.

I think the intent here was probably for this to affect all the models in this module? I would just define a base model `class NoProtectedBaseModel(BaseModle, protected_namespaces=())` (e.g https://docs.pydantic.dev/2.0/usage/model_config/#:~:text=class%20Model(BaseModel%2C%20extra%3D%27forbid%27)%3A) and inherit from that where applicable.